base/refpointer.jl

```diff
@@ -18,17 +18,6 @@ converted to a native pointer to the data it references.
 
 There is no invalid (NULL) `Ref` in Julia, but a `C_NULL` instance of `Ptr` can be passed to
 a `ccall` Ref argument.
-
-# Use in broadcasting
-
-Broadcasting with `Ref(x)` treats `x` as a scalar:
```

Comment:
I'd be inclined to keep this — it is common to see `Ref` in broadcast expressions and folks might `?Ref` to see/understand why.  I'd be okay with the status quo wording... but if you feel inspired you could make it a bit more descriptive instead of prescriptive.

Reply:
Good points!